housing_django/templates/home.html

```diff
@@ -3,10 +3,83 @@
 {% block content %}
 
 <div class="container">
-    <div class="row">
-        <div class="jumbotron">
-            
-        </div>
+  <div class="row">
+    <div class="jumbotron">
+      <button type="button" class="btn btn-success btn-sm create-btn">Create</button>
+      <button type="button" class="btn btn-info btn-sm export-btn export">Export report1</button>
+      <button type="button" class="btn btn-info btn-sm export-btn">Export report2</button>
+      <table class="table table-striped">
+        <tr>
+          <th>Firstname</th>
+          <th>Lastname</th>
+          <th>Address</th>
+          <th>Edit</th>
+          <th>Delete</th>
+        </tr>
+        <tr>
+          <td>Jim</td>
+          <td>Smith</td> 
+          <td>Redwood city</td>
+          <td><button class="btn btn-warning btn-xs" data-toggle="modal" data-target="#dataModal">Edit</button></td>
+          <td><button class="btn btn-danger btn-xs">Delete</button></td>
+        </tr>
+        <tr>
+          <td>Lily</td>
+          <td>Brown</td> 
+          <td>Union city</td>
+          <td><button class="btn btn-warning btn-xs">Edit</button></td>
+          <td><button class="btn btn-danger btn-xs">Delete</button></td>
+        </tr>
+        <tr>
+          <td>Henry</td>
+          <td>Dave</td> 
+          <td>Mountain view</td>
+          <td><button class="btn btn-warning btn-xs">Edit</button></td>
+          <td><button class="btn btn-danger btn-xs">Delete</button></td>
+        </tr>
+        <tr>
+          <td>Henry</td>
+          <td>Dave</td> 
+          <td>Mountain view</td>
+          <td><button class="btn btn-warning btn-xs">Edit</button></td>
+          <td><button class="btn btn-danger btn-xs">Delete</button></td>
+        </tr>
+      </table>
     </div>
+  </div>
+</div>
+<div class="modal fade" id="dataModal" tabindex="-1" role="dialog" aria-labelledby="dataModal" aria-hidden="true">
+  <div class="modal-dialog modal-lg">
+    <div class="modal-content">
```

Comment:
Can you move the form tag here? This way, the submit button can be inside the form tag.